Extensions/xkit_preferences.js

```diff
@@ -107,7 +107,7 @@ XKit.extensions.xkit_preferences = new Object({
 
 		//$("#new-xkit-control").tipTip({maxWidth: "auto", edgeOffset: 0, delay: 10 });
 
-		if(XKit.storage.get("xkit_preferences", "shown_welcome_bubble") !== "true") {
+		if(XKit.storage.get("xkit_preferences", "shown_welcome_bubble") !== "true" && XKit.interface.where().dashboard) {
 			XKit.extensions.xkit_preferences.show_welcome_bubble();
```

Comment:
That doesn't work if it's a string that gets returned. For a boolean you would be right